checks-superstaq/requirements.txt

```diff
@@ -6,6 +6,7 @@ isort[colors]>=5.10.1
 mypy>=1.0.0
 nbmake>=1.3.0
 nbsphinx>=0.9.0
+pydantic>=1.10.7,<2.0.0
```

Comment:
I believe so because of this: 
```
pydantic==1.10.13
└── qcs-api-client==0.21.6 [requires: pydantic>=1.7.2,<2.0.0]
    └── pyquil==3.2.1 [requires: qcs-api-client>=0.20.13,<0.22.0]
        └── cirq-rigetti==1.2.0 [requires: pyquil~=3.2.0]
            └── cirq==1.2.0 [requires: cirq-rigetti==1.2.0]
                └── cirq-superstaq==0.4.28 [requires: cirq>=1.0.0]
                    └── Supermarq==0.4.28 [requires: cirq-superstaq~=0.4.28]
```